Problema nas dependências do projeto ou no meu node version?

### Bom meus colegas, tentei rodar pela primeira vez o projeto e deu pau!

Queria ver se isso é algum BUG ou se é algum erro de instalação do meu node.
Minha versão do node é v14.17.3 e do docker compose é v2.1.1

### Usei esses dois comando apenas!
```cmd
npm install
npm run dev
```

**logs do npm install**
```cmd
npm install

npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...       
npm WARN old lockfile

added 1080 packages, and audited 1081 packages in 2m

96 packages are looking for funding
  run `npm fund` for details

4 vulnerabilities (1 high, 3 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

**logs do npm run dev**
```cmd
npm run dev   

> tabnews.com.br@1.0.0 dev
> kill-port 3000 && npm run services:up && npm run migration:run && npm run 
next && npm run services:stop

Process on port 3000 killed

> tabnews.com.br@1.0.0 services:up
> docker-compose -f infra/docker-compose.development.yml up -d

[+] Running 2/0
 - Container mailcatcher   Running                                     0.0s 
 - Container postgres-dev  Running                                     0.0s 

> tabnews.com.br@1.0.0 migration:run
> node infra/scripts/wait-for-db-connection-ready.js && node-pg-migrate up --envPath ./.env -m infra/migrations/ 2>migrations.log

internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'node:child_process'
Require stack:
- C:\Users\gaga1\OneDrive\Documentos\GitHub\tabnews.com.br\infra\scripts\wait-for-db-connection-ready.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)        
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\Users\gaga1\OneDrive\Documentos\GitHub\tabnews.com.br\infra\scripts\wait-for-db-connection-ready.js:1:18)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)        
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\gaga1\\OneDrive\\Documentos\\GitHub\\tabnews.com.br\\infra\\scripts\\wait-for-db-connection-ready.js'
  ]
}
```

**Além de alterar o lockfileVersion de 1 para 2 no package-lock.json**

Alguém poderia me informar, se eu estou fazendo algo errado ou se isso é um problema no projeto?

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Rodar o TabNews em sua máquina local é uma tarefa extremamente simples.
 
 Você precisa ter duas principais dependências instaladas:
 
-- Node.js 14.x (ou qualquer versão superior)
+- Node.js 14.18.3 (ou qualquer versão superior)
 - Docker Engine 17.12.0 com Docker Compose 1.24.1 (ou qualquer versão superior)
 
 Utiliza nvm? Então pode executar `nvm install` na pasta do projeto para instalar e utilizar a versão mais apropriada do Node.js.

